src/rank_llm/demo/rerank_dataset_with_prebuilt_index.py

```diff
@@ -24,3 +24,15 @@
 reranker = VicunaReranker()
 rerank_results = reranker.rerank(retrieved_results)
 print(rerank_results)
+
+from pathlib import Path
+from rank_llm.result import ResultsWriter
+
+# write rerank results
+writer = ResultsWriter(rerank_results)
+Path(f"demo_outputs/").mkdir(
```

Comment:
I created a dir named demo_outputs to store the three output files. Do you have any preference on how to store the files and/or names of the files?

Reply:
demo_outputs SG, please add it to gitignore though